heroku.yml

```diff
@@ -0,0 +1,9 @@
+build:
+  docker:
+    index-regen: heroku.Dockerfile
+run:
+  index-regen: python main.py
+release:
+  image: index-regen
+  command:
+    - PGSSLMODE=no-verify yarnpkg migrate
```

Comment:
PGSSLMODE was needed because without it the underlying node postgres library throws an error.
this tells the pg library under the hood "don't worry about verifying the ssl certificate, but still make use of it"
the node pg library expects you to add certificates to verify ssl connections.
we do this in the regen-server as well, seems fairly low stakes